s32193-SlawomirRudnicki/zadaniaTydzien3.txt

```diff
@@ -38,9 +42,14 @@ zrodlo -> manual
 
 3. wyswietlilem procesy bash:
 	ps -axu | grep bash
+	POPRAWIENIE:
+	ps -axu | grep -w bash -> powinno znalezc tylko linie w ktorych jest zawarty string "bash" jako calosc, a nie "bash" rowniez jako czesc stringu, 
```

Comment:
Lepiej acz nie o to chodziło. Jest opcja to szukania po nazwie polecenia wewnątrz samego polecenia ps. Poszukaj w manualu.
Ale faktycznie jest to ciekawe rozwiązanie, które zaproponowałeś.